instance/default.cfg

```diff
@@ -11,6 +11,7 @@ CELERY_RESULT_BACKEND = 'redis://localhost:6379/0'
 CELERY_BROKER_URL = 'redis://localhost:6379/0'
 ADMINS = ['deepshikhaaa']
 REGION_NAME = 'us-east-2'
-IMAGE_ID = '<ami_image_id>'
+IMAGE_ID_7 = '<ami_image_id>'
+IMAGE_ID_8 = '<ami_image_id>' #Please do change it accordingly
```

Comment:
Can you please change the '<ami_image_id>' to <ami_image_id_centos7> and so on..